### PR TITLE
Add mb (make-bucket) command

### LINKED ISSUE #25: Is there no option to create buckets?
Something like `s5cmd mb s3://bucketname` is missing.

It is quite essential to be able to create buckets.

### PR DESCRIPTION
**Pull request by ilkinulas**
- added mb (make bucket) command

### CODE REVIEW

File: core/job_exec.go [hunk 1]
```diff
@@ -324,3 +324,19 @@ func Size(ctx context.Context, job *Job) *JobResponse {
...
+	client, err := storage.NewClient(bucket)
+	if err != nil {
+		return jobResponse(err)
+	}
+
+	err = client.MakeBucket(ctx, bucket.Bucket)
+	if err != nil {
+		return jobResponse(err)
+	}
+	log.Logger.Success(fmt.Sprintf("Bucket '%s' created successfully.", bucket))
```
Comment (igungor) [inline core/job_exec.go]: `Success` supports Printf style formatting.

File: e2e/mb_test.go [hunk 2]
```diff
@@ -0,0 +1,38 @@
+package e2e
+
+import (
+	"fmt"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/s3"
+	"gotest.tools/v3/icmd"
+	"testing"
```
Comment (igungor) [inline e2e/mb_test.go]: please run goimports.

File: opt/opt.go [hunk 3]
```diff
@@ -160,6 +160,7 @@ type ParamType int
 const (
 	Unchecked          ParamType = iota // Arbitrary single parameter
 	UncheckedOneOrMore                  // One or more arbitrary parameters (special case)
+	S3Bucket                            //Bucket
```
Comment (igungor) [inline opt/opt.go]: // Bucket (whitespace)

File: storage/s3.go [hunk 4]
```diff
@@ -409,6 +409,18 @@ func (s *S3) ListBuckets(ctx context.Context, prefix string) ([]Bucket, error) {
...
 
+// MakeBucket creates an S3 bucket with the given name.
+func (s *S3) MakeBucket(ctx context.Context, name string) error {
+	o, err := s.api.CreateBucketWithContext(ctx, &s3.CreateBucketInput{
+		Bucket: aws.String(name),
+	})
+	if err != nil {
+		return err
+	}
+	fmt.Printf("make bucket output %v\n", o)
```
Comment (igungor) [inline storage/s3.go]: remove

File: storage/s3.go [hunk 5]
```diff
@@ -409,6 +409,17 @@ func (s *S3) ListBuckets(ctx context.Context, prefix string) ([]Bucket, error) {
 	return buckets, nil
 }
 
+// MakeBucket creates an S3 bucket with the given name.
+func (s *S3) MakeBucket(ctx context.Context, name string) error {
+	_, err := s.api.CreateBucketWithContext(ctx, &s3.CreateBucketInput{
+		Bucket: aws.String(name),
+	})
+	if err != nil {
```
Comment (sonmezonur) [inline storage/s3.go]: we can directly return err here.

**Pull request merged**